README.md

```diff
@@ -1,31 +1,156 @@
-# Angular-200
+# Angular 200
+by [SFEIR School](https://www.sfeir.com/formation/school/)
 
-This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.20-4.
+Angular est devenu un acteur incontournable de l'écosystème Front. Développé par Google ce framework a su s’imposer pour devenir la référence. Mais l’annonce de la version 2 a boulversé la donne et a donné lieu à des débats “animés”. L’objectif de cette formation est de vous présenter les nouvelles orientations d’Angular 2. Pour cela nous alternons de façon équilibrée les phases d’apprentissage théorique et les exercices pratiques sur un projet réaliste.
 
-## Development server
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
-## Code scaffolding
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive/pipe/service/class`.
 
-## Build
+![logo](https://www.sfeir.com/img/school/formations/Angular%20200.png)
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+[Calendrier des formations, liste des formateurs et programme de formation](https://www.sfeir.com/formation/school/angular-200/)
 
-## Running unit tests
+## Table des matières
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+  * [Support de formation](https://github.com/Sfeir/angular2-200/blob/step-01/support.pdf)
+  * [Installation et lancement](#Installation-et-lancement)
+  * [Déroulement de la formation](#déroulement-de-la-formation)
+  * [Planning de la formation](#Planning-de-la-formation)
+  * [Liste des projets](#Liste-des-projets)
+  * [Stack technique](#stack-technique)
 
-## Running end-to-end tests
+## Installation et lancement
 
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/). 
-Before running the tests make sure you are serving the app via `ng serve`.
+  * `git clone https://github.com/Sfeir/angular-200.git`
+  * `npm install -g @angular/cli` 
+  * `cd angular-200`
+  * `npm install`
 
-## Deploying to Github Pages
 
-Run `ng github-pages:deploy` to deploy to Github Pages.
+  * `npm run server`
 
-## Further help
 
-To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+  
+
+## Déroulement de la formation
+Un TP :
+
+  * un projet par exercice :
+`npm run client step01`
+  * un projet par solution :
+`npm run client step01solution`
+
+ 
+## Planning de la formation
```

Comment:
Please remove this part, it would be to much to maintain for now.

Reply:
As i'm seeing 2 sections, are you talking about the planning or the script commands for steps?
I deleted script commands meanwhile but will make changes depending on your answer.

Reply:
The planning part. Thx

Reply:
i removed it and added back the script commands section i deleted meanwhile.